application/__init__.py

```
@@ -26,15 +26,22 @@
+logger = logging.getLogger(__name__)
+
 try:
     with open('log_config.yaml', 'r') as config_file:
         config = yaml.safe_load(config_file.read())
     logging.config.dictConfig(config)
-except Exception as ex:
-    print("Logging setup failed with exception = ", ex)
-
-logger = logging.getLogger(__name__)
-logger.warning(f"Logging is set up with config={config}")
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)